### PR TITLE
Add ability to hide scroll indicators

### PR DESCRIPTION
**Pull request by DannyCamenisch**
Thanks for this awesome packet! I added the ability to hide the scroll indicator. To not break existing code, I made it a view modifier `.hiddeIndicators()`.

### CODE REVIEW

File: Source/ScalingHeaderScrollView.swift [hunk 1]
```diff
@@ -59,6 +59,9 @@ public struct ScalingHeaderScrollView<Header: View, Content: View>: View {
     /// Allow force snap to closest position after lifting the finger, i.e. forbid to be left in unfinished state
     private var allowsHeaderSnapFlag: Bool = false
     
+    /// Shows or hidde the indictor for the scrollview
```
Comment (f3dm76) [inline Source/ScalingHeaderScrollView.swift:62]: There are some typos, would you please use this phrase?
Shows or hides the indicator for the scrollView

File: Source/ScalingHeaderScrollView.swift [hunk 2]
```diff
@@ -274,4 +277,11 @@ extension ScalingHeaderScrollView {
         scalingHeaderScrollView.allowsHeaderSnapFlag = true
         return scalingHeaderScrollView
     }
+    
+    /// Hiddes scroll indicators
+    public func hiddeIndicators() -> ScalingHeaderScrollView {
```
Comment (f3dm76) [inline Source/ScalingHeaderScrollView.swift:282]: could you please name it `hideScrollIndicators`?

**Review by f3dm76 — CHANGES_REQUESTED** [review]

File: Source/ScalingHeaderScrollView.swift (hunk 1 above)
Comment (DannyCamenisch) [reply Source/ScalingHeaderScrollView.swift]: done

**Comment by DannyCamenisch** [comment]
Hey @f3dm76 I made the requested changes. Have a nice day!

**Pull request merged**